docs/build/configure/webhooks.md

```diff
@@ -11,35 +11,91 @@ Webhooks allow you to trigger actions when certain types of data are sent from y
+- Data has been synced to the cloud: trigger when data from the machine is synced
+- Part is online: trigger continuously at specified interval while the {{< glossary_tooltip term_id="part" text="machine part" >}} is online
+- Part is offline: trigger continuously at specified interval while the machine part is offline
 
 To configure a webhook:
 
-{{< tabs name="Types of Webhooks" >}}
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
 
 1. Go to the **CONFIGURE** tab of your machine on the [Viam app](https://app.viam.com).
-   Select **JSON** mode.
+   Click the **+** (Create) button in the left side menu and select **Webhook**.
+
+   {{<imgproc src="/build/configure/webhook-create.png" resize="x400" declaredimensions=true alt="The Create menu with Webhook at the bottom of the list of options." >}}
+
+2. Name the webhook and click **Create**.
+
+3. Select the type of event to trigger on from the **Type** dropdown.
+
+4. Follow the instructions depending on the type of webhook you want to implement:
+
+{{< tabs name="Types of Webhooks" >}}
+{{% tab name="Data synced to cloud" %}}
 
-2. Follow the instructions depending on the type of webhook you want to implement:
+5. Select the types of data you want to trigger on from the dropdown.
 
-{{% tab name="part_data_ingested" %}}
+{{% alert title="Note" color="note" %}}
+Be sure to configure [data capture](/data/capture/) and [cloud sync](/data/cloud-sync/) for the relevant components.
+For example, if you want to trigger a webhook on temperature readings, configure data capture and sync on your temperature sensor.
+Be aware that the component must return the type of data you configured in the webhook's **Data Types**.
+{{% /alert %}}
 
-1. Paste the following JSON template into your JSON config.
-   `"webhooks"` is a top-level section like `"components"`, `"services"`, or any of the other config sections.
+{{% /tab %}}
+{{% tab name="Part is online" %}}
+
+5. While your part is online, the webhook action triggers at a specified interval.
+   Edit the **Time between notifications** attribute to set this interval according to your preferences.
+
+{{% /tab %}}
+{{% tab name="Part is offline" %}}
+
+5. While your part is offline, the webhook action triggers at a specified interval.
+   Edit the **Time between notifications** attribute to set this interval according to your preferences.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+6. Replace the URL value with the URL of your cloud/lambda function.
+
+   ![The webhook configured with an example URL in the Viam app.](/build/configure/webhook-configured.png)
+
+{{% /tab %}}
+{{% tab name="Raw JSON" %}}
+
+If you prefer to configure your webhook with raw JSON instead of the config builder, you can paste one of the following JSON templates into your JSON config.
+`"webhooks"` is a top-level section like `"components"`, `"services"`, or any of the other config sections.
 
 {{< tabs >}}
-{{% tab name="JSON Template" %}}
+{{% tab name="JSON Template: Data" %}}
```

Comment:
```suggestion
{{% tab name="JSON Template: Data has been synced to the cloud" %}}
```
Maybe clunky but I think they should match, or maybe you could make "type" clear in intro and use that here?

Reply:
These tab names with the "JSON Template" are already so long I don't want to make them even longer such that they have to wrap onto a new line on narrow screens (which I think makes it look confusing). I'll add this:
```suggestion
{{% tab name="JSON Template: Data Synced" %}}
```